components/library/mozilla-nss/Makefile

```diff
@@ -51,9 +51,9 @@ COMPONENT_TEST_ENV += $(COMPONENT_BUILD_ENV)
 COMPONENT_TEST_ENV += HOST=127.0.0.1
 COMPONENT_TEST_DIR = $(BUILD_DIR_$(BITS))/nss/tests
 
-include $(WS_TOP)/make-rules/prep.mk
-include $(WS_TOP)/make-rules/justmake.mk
-include $(WS_TOP)/make-rules/ips.mk
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/justmake.mk
+include $(WS_MAKE_RULES)/ips.mk
```

Comment:
Can we use `include $(WS_MAKE_RULES)/common.mk` and `BUILD_STYLE=justmake`?

Reply:
I have tried that, but with it the build failed for me. So I left most of the Makefile untouched...